### ISSUE TITLE
Implement VBS Standard Library

### ISSUE THREAD

**Issue by freezy** [vpdb/vpx-js#58]
VBScript obviously comes with a standard lib. This is probably one of the last tasks of the scripting engine. The goal is not to have a 100% covered stdlib, but to analyze the table scripts and check which functions and classes are used.

**Comment by neophob** [comment]
@freezy you worked and merged this, right?

**Comment by freezy** [comment]
Yes, some of the stdlib is done but there's still a lot more to cover. I propose to close this when we have `core.vbs` transpiled correctly.
